### PR TITLE
Add PLBartTokenizerFast

### PR DESCRIPTION
**Pull request by gchhablani**
This PR adds fast tokenizer for PLBart. It is currently a work-in-progress.

@patil-suraj @sgugger @LysandreJik @patrickvonplaten 

### To-do:
- [ ] Add missing classes convert_slow_tokenizer
- [ ] Convert and add the tokenizer files to all the model checkpoints
- [ ] Fix the tests

### CODE REVIEW

File: tests/plbart/test_tokenization_plbart_fast.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            "\u0120l",
+            "\u0120n",
+            "\u0120lo",
+            "\u0120low",
+            "er",
+            "\u0120lowest",
+            "\u0120newer",
+            "\u0120wider",
+            "<unk>",
+        ]
```
Comment (sgugger) [inline tests/plbart/test_tokenization_plbart_fast.py:57]: Should be put on one line, with the formatter being disabled just for the line.

**Review by sgugger — COMMENTED** [review]
Thanks for adding this! I think a converter should also be added in `convert_slow_tokenizers_checkpoints_to_fast.py` for your new fast tokenizer to work.

**Comment by gchhablani** [comment]
@sgugger I am actually facing an issue for the converter. The `PLBartTokenizer` accepts an argument called `language_codes` based on which the language codes for the tokenizer are decider. 

The converter only takes one list of language codes. 

Should I modify the method which is used in `PreTrainedTokenizerFast` to allow for adding extra arguments like `language_codes`? What is the best way of going about this according to you?

**Comment by gchhablani** [comment]
Unstale

**Comment by patil-suraj** [comment]
Hey @gchhablani ! Let us know if you need any help with this :)

**Comment by FahadEbrahim** [comment]
Any updates regarding the PLBART fast tokenizer?

**Pull request closed without merging**